examples/bpk-component-navigation-tab-group/examples.tsx

```diff
@@ -92,7 +92,6 @@ const SimpleCanvasDefault = () => (
       onItemClick={() => {}}
       selectedIndex={0}
       type={NAVIGATION_TAB_GROUP_TYPES.CanvasDefault}
-      ariaLabel="Navigation tabs"
```

Comment:
Why did you decided to remove the ariaLabel?

Reply:
Not remove,  Just change ariaLabel as optional. And this example is used for test props without ariaLabel

Reply:
When I check header tabs that is not include ariaLabel,so I think it is need to change to option. Is it reasonable for cosumer decide to whether pass this ariaLabel?

Reply:
In this case, I think it's best practice for accessibility standards to not have it as optional

Reply:
@olliecurtis @amburrrshand Thank you for your advice. I have reverted the ariaLabel as required.